MyShopApp/MyShopApp/View/ProductsView.swift

```diff
@@ -9,20 +9,130 @@ import UIKit
 
 /// List of our products
 final class ProductsViews: UIView {
-
+    
+    // MARK: - Properties
     lazy var productLable: UILabel = {
         let label = UILabel()
-        label.frame = CGRect(x: 115, y: 100, width: 200, height: 100)
+        label.frame = CGRect(x: 0, y: 100, width: 400, height: 20)
+        label.textAlignment = .center
+        label.font = .systemFont(ofSize: 18, weight: .bold)
         label.numberOfLines = 0
         label.textColor = .label
         return label
     }()
+    
+    lazy var productSecondLabel: UILabel = {
+        let label = UILabel()
+        label.frame = CGRect(x: 0, y: 480, width: 400, height: 20)
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.font = .systemFont(ofSize: 15)
+        label.textColor = .white
+        return label
+    }()
 
     lazy var productImageView: UIImageView = {
         let image = UIImageView()
-        image.frame = CGRect(x: 115, y: 300, width: 200, height: 200)
+        image.frame = CGRect(x: 50, y: 0, width: 300, height: 300)
         image.contentMode = .scaleAspectFit
         return image
     }()
-
+    
+    lazy var priceLabel: UILabel = {
+        let label = UILabel()
+        label.frame = CGRect(x: 0, y: 120, width: 400, height: 30)
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.textColor = .gray
+        return label
+    }()
+    
+    lazy var firstColorButton: UIButton = {
+        let button = UIButton()
+        button.frame = CGRect(x: 170, y: 520, width: 36, height: 36)
+        button.layer.cornerRadius = 18
+        button.layer.borderWidth = 3
+        button.backgroundColor = UIColor(named: "Ernest")
+        return button
+    }()
+    
+    lazy var secondColorButton: UIButton = {
+        let button = UIButton()
+        button.frame = CGRect(x: 215, y: 520, width: 36, height: 36)
+        button.layer.cornerRadius = 18
+        button.layer.borderWidth = 2
+        button.backgroundColor = UIColor(named: "ErnestSecond")
+        button.layer.borderColor = UIColor.systemBlue.cgColor
+        return button
+    }()
+    
+    lazy var orderButton: UIButton = {
+        let button = UIButton()
+        button.frame = CGRect(x: 35, y: 665, width: 350, height: 45)
+        button.setTitle("Add to cart", for: .normal)
```

Comment:
```suggestion
        button.setTitle("", for: .normal)
```
Constants
все текстовки в константы